src/main/java/com/github/fabienrenaud/jjb/databind/Deserialization.java

```diff
@@ -153,6 +153,6 @@ public Object quickbuf_json() throws Exception {
     @Benchmark
     @Override
     public Object wast() throws Exception {
-        return io.github.wycst.wast.json.JSON.parseObject(JSON_SOURCE().nextString(), JSON_SOURCE().pojoType(), ReadOption.UseJDKDoubleParser);
+        return io.github.wycst.wast.json.JSON.parseObject(JSON_SOURCE().nextString(), JSON_SOURCE().pojoType()/*, ReadOption.UseJDKDoubleParser*/);
```

Comment:
Either:
 - remove entirely the `ReadOption.UseJDKDoubleParse` part to show/suggest it's the normal default setup;
 - or, add a comment explaining why this is kept here but commented.